src/openrct2/ride/Ride.cpp

```diff
@@ -835,12 +835,11 @@ void Ride::FormatStatusTo(Formatter& ft) const
     {
         ft.Add<rct_string_id>(STR_TEST_RUN);
     }
-    else if (
-        mode == RIDE_MODE_RACE && !(lifecycle_flags & RIDE_LIFECYCLE_PASS_STATION_NO_STOPPING))
+    else if (mode == RIDE_MODE_RACE && !(lifecycle_flags & RIDE_LIFECYCLE_PASS_STATION_NO_STOPPING))
     {
         auto peep = GetEntity<Peep>(race_winner);
-            ft.Add<rct_string_id>(STR_RACE_WON_BY);
-            peep->FormatNameTo(ft);
+        ft.Add<rct_string_id>(STR_RACE_WON_BY);
```

Comment:
Sorry I think I might have not been very clear. I meant you don't need both nullptr checks. So could you readd one of them.

Reply:
I've restored the check.
Everything is ok :) I could have asked or read what you have written more carefully.